### PR TITLE
Improved cursor position display for light mode

### PR DESCRIPTION
**Pull request by jeremy-rifkin**
Improved style for light mode:

image

### CODE REVIEW

File: static/themes/default-theme.scss [hunk 1]
```diff
@@ -331,4 +331,10 @@ div.argmenuitem span.argdescription {
 
 .ts-dropdown .optgroup-header {
     background-color: #f6f6f6;
- }
+}
+
+.currentCursorPosition {
+    color: #15a3b9;
```
Comment (RubenRBS) [inline static/themes/default-theme.scss:337]: Contrast is too low in the light theme, this might be better being a darker gray / outright black

**Review by RubenRBS — APPROVED** [review]

File: static/themes/default-theme.scss (hunk 1 above)
Comment (mattgodbolt) [reply static/themes/default-theme.scss:337]: Do you think it's ok to merge though Ruben? You're our colour expert :-)

Comment (RubenRBS) [reply static/themes/default-theme.scss:337]: Yes! By all means

**Comment by mattgodbolt** [comment]
oh cool! thanks ruben!

**Pull request merged**